inginious/frontend/templates/course_admin/task_dispensers/task_buttons.html

```diff
@@ -2,6 +2,9 @@
 {# more information about the licensing of this file. #}
 
 <div class="ml-auto btn-group btn-group-sm" role="group">
+    <button class="btn btn-primary" data-toggle="modal" data-target="#weightModal_{{taskid}}">
+        Weight
```

Comment:
You can use a gear icon or something similar as well as a tooltip to mimick the other buttons and to indicate additional settings.